### DISCUSSION TITLE
Adjust proc:Program column width

### DISCUSSION THREAD

**Discussion by webbp** [aristocratos/bpytop#284] [Q&A] [answered]
bpytop is my new favorite thing. Thanks for making it!

I have a minor question: is there a way to adjust proc:Program column width? In tree view with a wide proc pane, e.g., if I hide mem and net, the program name is truncated to 15 chars, yet there is about 100 chars of empty space after them. In normal view with a wide proc pane, Program is truncated to 14 chars while Arguments gets about 120 chars.

Many thanks,
Webb

**Comment by aristocratos** [comment, marked as answer]
@webbp 
Customizable process columns (size and content) is in the plans.
But I sadly don't have much time at the moment to work on BpyTop, will hopefully change in the coming months.
